### PR TITLE
GDAL try to fix libcurl dependency on Linux

### PR DESCRIPTION
**Pull request by visr**
By adding -lstdc++ as was done for GEOS.

Hopefully this will fix the last part of #418:
```
found libcurl version 7.66.0
checking for curl_global_init in -lcurl... no

cURL support (wms/wcs/...):no
```

### CODE REVIEW

**Comment by visr** [comment]
Ok I guess -lstdc++ was not the (only?) missing library here.

File: G/GDAL/build_tarballs.jl [hunk 1]
```diff
@@ -24,6 +24,7 @@ elif [[ "${target}" == *-linux-* ]]; then
     # Make sure GEOS is linked against libstdc++
     atomic_patch -p1 "$WORKSPACE/srcdir/patches/geos-m4-extra-libs.patch"
```
Comment (giordano) [inline G/GDAL/build_tarballs.jl]: > Ok I guess -lstdc++ was not the (only?) missing library here.

Applying the patch might help :wink: 

```suggestion
     atomic_patch -p1 "$WORKSPACE/srcdir/patches/geos-m4-extra-libs.patch"
     atomic_patch -p1 "$WORKSPACE/srcdir/patches/configure_ac_curl_libs.patch"
```

Comment (visr) [reply G/GDAL/build_tarballs.jl]: :sweat_smile:

**Comment by giordano** [comment]
Any other library you want to check? :slightly_smiling_face:

**Comment by visr** [comment]
Ha I think we have them all. This was the last error that was caught in the unit tests in https://github.com/JuliaGeo/GDAL.jl/pull/82 (which of course have less platform coverage than tests here).

**Pull request merged**